### PR TITLE
drop support for internal modules

### PR DESCRIPTION
**Pull request by chirag04**

### CODE REVIEW

**Comment by taion** [comment]
Would it be crazy to check the Lerna config?

**Comment by chirag04** [comment]
make no assumptions? not every project will be a Lerna project?

File: index.js [hunk 1]
```diff
@@ -10,16 +10,12 @@ const aliased = [
...
-function isInternalModule({ moduleName }) {
-  return moduleName.startsWith('@qsi/');
-}
-
 function isAliasedModule({ moduleName }) {
   return aliased.some(n => moduleName.startsWith(n));
 }
 
 function isCssModule({ moduleName }) {
-  return moduleName.includes('@qsi/ui-theme') || moduleName.includes('.css');
```
Comment (chirag04) [inline index.js:22]: @jquense this is a bit extreme and i'm happy to revert. upto you.

**Comment by chirag04** [comment]
npm test output:

```
/* eslint-disable */

import getNodes from '@qsi/common/lib/getNodes';
import PageForm from '@qsi/ui-app/lib/PageForm';
import theme from '@qsi/ui-theme'
import Card from '@qsi/ui/lib/Card';
import Fieldset from '@qsi/ui/lib/Fieldset';
import Form from '@qsi/ui/lib/RelayForm';
import { omitExtraProps } from '@qsi/ui/lib/utils/Props';
import PropTypes from 'prop-types';
import React from 'react';
import { remove } from 'react-formal/lib/util/ErrorUtils';
import Relay from 'react-relay';

import EntityFormSection from 'components/EntityFormSection';
import ParamsFormSection from 'components/ParamsFormSection';
import WorkflowInput, { fragments as inputFragments }
  from 'components/WorkflowInput';
import messages from 'messages/analysis';
import workflowMessages from 'messages/analysisWorkflow';
import fetchWorkflowParams from 'utils/fetchWorkflowParams';
import analysisSchema, { deserialize, serialize, setParams, setWorkflow }
  from '../schemas/analysis';
import SampleAssociationFormSection, { fragments as sampleAssocFragments }
  from './SampleAssociationFormSection';

import './styles.css';
import otherStyles from '@qsi/ui/styles/text.css';
import styles from './foo.css';
```

**Comment by taion** [comment]
I dunno, I think maybe it should be configurable. I do like the idea of grouping "internal" imports in a multi-project differently, maybe.

**Comment by jquense** [comment]
I'm fine with it not being configurable I think.

I guess the actual value in the grouping is in knowing the sort and location of modules right? If something is in node_modules it doesn't matter that much that it's symlinked right?

File: index.js (hunk 1 above)
Comment (jquense) [reply index.js:22]: I think it's fine to remove, its not even actually css it's javascript exported here.

**Comment by chirag04** [comment]
g2g?

**Comment by jquense** [comment]
ugh maybe I am changing my mind, it's not great that all these imports are first, I think it obscures the things that are "real" dependencies (e.g. not owned by us) and stuff that is merely organizationally seperate? I realize tho i have a different relationship with these modules then you do @chirag04 

Would it a reasonable to make the `@scope` configurable? Even if we sort the @scoped modules below the node/external ones?

**Comment by taion** [comment]
I think either scope should be configurable, or we should automatically infer Lerna packages if applicable.

**Comment by jquense** [comment]
to be clear I don't group them based on whether they are part of the lerna repo, I group _all_ '@qsi' packages that way regardless of the app :P

**Comment by taion** [comment]
But if hypothetically there were one that weren't in the monorepo? Would you want it grouped with "internal" or "external"?

**Comment by jquense** [comment]
we do have ones that aren't (the sunburst mainly), and they are all external from the perspective of the Admin, Offline, and Factory apps, which are different repos. I jsut group them all together as "internal".

I think it makes sense to group them with external but i'd want to sort the scoped packages under the node and npm ones we don't own e.g.

```
found
react
@qsi

./foo
```

**Pull request closed without merging**